client/layout/global-sidebar/menu-items/notifications/index.jsx

```diff
@@ -169,27 +170,31 @@ class SidebarNotifications extends Component {
 						</DismissibleCard>,
 						document.querySelector( '.layout' )
 					) }
-				<SidebarMenuItem
-					url="/notifications"
-					icon={ <BellIcon newItems={ this.state.newNote } active={ this.props.isActive } /> }
-					onClick={ this.handleClick }
-					isActive={ this.props.isActive }
-					tooltip={ this.props.tooltip }
-					tooltipPlacement="top"
-					className={ classes }
-					ref={ this.notificationLink }
-					key={ this.state.animationState }
-				/>
-				<div className="sidebar-notifications__panel" ref={ this.notificationPanel }>
-					<AsyncLoad
-						require="calypso/notifications"
-						isShowing={ this.props.isNotificationsOpen }
-						checkToggle={ this.checkToggleNotes }
-						setIndicator={ this.setNotesIndicator }
-						isGlobalSidebarVisible={ true }
-						placeholder={ null }
+				{ this.props.masterbarIsHidden && (
```

Comment:
Did you mean:
```suggestion
				{ ! this.props.masterbarIsHidden && (
```
Otherwise, the desktop view would no longer have the notification icon.

![Screenshot 2024-05-14 at 1 56 46 PM](https://github.com/Automattic/wp-calypso/assets/797888/11770fda-0e85-4ec6-8839-b5c479cb0f0d)

Reply:
No. It should show only when the masterbar is hidden. But the condition doesn't seem enough...